core/src/main/java/io/confluent/kafka/schemaregistry/storage/KafkaStoreReaderThread.java

```diff
@@ -208,7 +200,9 @@ public void waitUntilOffset(long offset, long timeout, TimeUnit timeUnit) throws
     if (offset < 0) {
       throw new StoreException("KafkaStoreReaderThread can't wait for a negative offset.");
     }
-    
+
+    log.trace("Waiting for offset to be " + offset + ". Currently " + offsetInSchemasTopic);
```

Comment:
Nitpick: This is a little nicer:

```java
log.trace("Waiting for offset to be {}. Currently {}", offset, offsetInSchemasTopic);
```
Also, the message is a little unclear. Maybe something like "Waiting to read to offset {}"?